core/lexicon/en/tv_widget.inc.php

```diff
@@ -7,50 +7,94 @@
+$_lang['combo_listempty_text_desc'] = 'Message to display when typed text does not match existing options.';
 $_lang['combo_listheight'] = 'List Height';
 $_lang['combo_listheight_desc'] = 'The height, in % or px, of the dropdown list itself. Defaults to the height of the combobox.';
 $_lang['combo_listwidth'] = 'List Width';
 $_lang['combo_listwidth_desc'] = 'The width, in % or px, of the dropdown list itself. Defaults to the width of the combobox.';
 $_lang['combo_maxheight'] = 'Max Height';
-$_lang['combo_maxheight_desc'] = 'The maximum height in pixels of the dropdown list before scrollbars are shown (defaults to 300).';
+$_lang['combo_maxheight_desc'] = 'The maximum height in pixels of the dropdown list before scrollbars are shown. (Default: 300)';
 $_lang['combo_stackitems'] = 'Stack Selected Items';
 $_lang['combo_stackitems_desc'] = 'When set to Yes, the items will be stacked 1 per line. Defaults to No, which displays the items inline.';
 $_lang['combo_title'] = 'List Header';
 $_lang['combo_title_desc'] = 'If supplied, a header element is created containing this text and added into the top of the dropdown list.';
 $_lang['combo_typeahead'] = 'Enable Type-Ahead';
-$_lang['combo_typeahead_desc'] = 'If yes, populate and autoselect the remainder of the text being typed after a configurable delay (Type-Ahead Delay) if it matches a known value (defaults to off).';
-$_lang['combo_typeahead_delay'] = 'Type-Ahead Delay';
-$_lang['combo_typeahead_delay_desc'] = 'The length of time in milliseconds to wait until the Type-Ahead text is displayed if Type-Ahead is enabled (defaults to 250).';
+$_lang['combo_typeahead_desc'] = 'Populate and autoselect options that match as you type after a configurable delay. (Default: No)';
+$_lang['combo_typeahead_delay'] = 'Delay';
+$_lang['combo_typeahead_delay_desc'] = 'Milliseconds before a matched option is shown. (Default: 250)';
 $_lang['date'] = 'Date';
 $_lang['date_format'] = 'Date Format';
-$_lang['date_use_current'] = 'If no value, use current date';
+$_lang['date_format_desc'] = 'Enter a format using <a href="https://www.php.net/strftime" target="_blank">php’s strftime syntax</a>.
+    <div class="example-list">Common examples include:
+        <ul>
+            <li><span class="example-input">%A %d, %B %Y</span> ('.strftime("%A %d, %B %Y").') (default format)</li>
+            <li><span class="example-input">%a, %b %e, %Y</span> ('.strftime("%a, %b %e, %Y").')</li>
+            <li><span class="example-input">%m/%d/%Y</span> ('.strftime("%m/%d/%Y").')</li>
+            <li><span class="example-input">%Y-%m-%d</span> ('.strftime("%Y-%m-%d").')</li>
+            <li><span class="example-input">%Y-%m-%d %T</span> ('.strftime("%Y-%m-%d %T").')</li>
+            <li><span class="example-input">%b %e, %Y</span> ('.strftime("%b %e, %Y").')</li>
+            <li><span class="example-input">%e %h %Y %l:%M %p</span> ('.strftime("%e %h %Y %l:%M %p").')</li>
+        </ul>
+    </div>';
```

Comment:
If you haven't, I think you might want to run this and see what it does before making final judgement. I think the feature added is quite helpful in getting started with the code examples and being able to see them much more clearly when inserted into the field. I do have translations in mind, and wouldn't opt to do this all over the place. Think about it...

Reply:
I'm worried this logic will either get broken by crowdin, or will not work as you expect it to because the lexicon is cached. The idea of showing the actual time as how it would render with that option is nice, though. 

(Also I don't really care too much about HTML in lexicons if it serves a good purpose.)

Reply:
Oh man, these clickable examples do look _so freaking awesome_ in the manager. Nice work!

Reply:
One way to get the logic out of the lexicons is to pass it into the lexicon with placeholders... render out the examples server side, pass it into the lexicon. That way it could still be dynamic but we don't have any logic code in the lexicon files.

Reply:
OK, I'll rework the lexes a bit to get the php out of them...

Reply:
OK, I made a number of changes in the latest commit (fda8f41) addressing this. Now all example logic/content gets inserted via placeholders now.